pkg/apis/client/v1alpha1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+// +groupName=client.knative.dev
+
+// Package v1alpha1 is the v1alpha1 version of the API.
+package v1alpha1 // import "knative.dev/client/pkg/apis/client/serving/v1alpha1"
```

Comment:
Can we remove the commented `import...` ?

Reply:
I think this is auto-generated code, so we don't have direct influence on the generated part. Only how we start the code generation, so I think this is ok.